apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,9 @@
 # API Changelog
 
+## BFD-8: Add and map coverage fields dual01..dual12 and rfrnc_yr to the coverage resource
+
+The dual_01..dual12 and rfnrc_yr are essentially coverage related fields and need to be included in the coverage resource. Previously, these fields were only available on the patient resource and when beneficiaries choose to redact their demographic data, downstream consumers lose the these vital eligibility fields thus the need to add them to the coverage resource. In future versions of the API, we'll want to consider removing these from the patient resource. 
```

Comment:
Recommend correcting wording... `consumers lose the these vital eligibility fields` :)

Reply:
Fixed